src/MOI_wrapper.jl

```diff
@@ -431,5 +433,79 @@ end
     end
     # qmatind is ::Vector{Cint}, so we convert back to ::Vector{Int}.
     return sparse(Int.(qmatind), qmatcol, qmatval)
- end
+end
+
+function compute_conflict(model::Optimizer)
+    model.conflict = refineconflict(model.inner)
+    return
+end
 
+"""
+    ConflictStatus()
+
+An optimizer attribute indicating the status of the last computed conflict.
+"""
+struct ConflictStatus <: MOI.AbstractOptimizerAttribute  end
+MOI.is_set_by_optimize(::ConflictStatus) = true
+
+function MOI.get(model::Optimizer, attribute::ConflictStatus) 
```

Comment:
We shouldn't rely on the user calling `compute_conflict`. It is easier if we do something like:
```julia
function MOI.get(model::Optimizer, ::ConflictStatus) 
    if model.conflict === nothing
        model.conflict = c_api_getconflict(model.inner)
    end
    if model.conflict.stat == 
```

Reply:
We may need to think about checking that the termination status is infeasible, and under what circumstances it's actually okay to call `c_api_getconflict`.